src/main/java/software/amazon/qldb/exceptions/ExecuteException.java

```diff
@@ -21,17 +21,37 @@
  * <p><b>Note</b>: this class is for internal use only.</p>
  */
 public class ExecuteException extends RuntimeException {
-    public SdkException cause;
-    public boolean isRetriable;
-    public boolean isAborted;
-    public boolean isISE;
-    public String txnId;
+    private final SdkException cause;
+    private final boolean retryable;
+    private final boolean aborted;
+    private final boolean ise;
+    private final String txnId;
 
-    public ExecuteException(SdkException cause, boolean isRetriable, boolean isAborted, boolean isISE, String txnId) {
+    public ExecuteException(SdkException cause, boolean isRetryable, boolean isAborted, boolean isISE, String txnId) {
         this.cause = cause;
-        this.isRetriable = isRetriable;
-        this.isAborted = isAborted;
-        this.isISE = isISE;
+        this.retryable = isRetryable;
+        this.aborted = isAborted;
+        this.ise = isISE;
         this.txnId = txnId;
     }
+
+    public SdkException getCause() {
+        return cause;
+    }
+
+    public boolean isRetryable() {
+        return retryable;
+    }
+
+    public boolean isAborted() {
+        return aborted;
+    }
+
+    public String getTxnId() {
+        return txnId;
+    }
+
+    public boolean isISE() {
```

Comment:
Expand this acronym

Reply:
Updated to `isInvalidSessionException()`